packages/web-api/src/controllerDispatcher.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Context } from '@azure/functions';
+import { Container } from 'inversify';
+import { BaseTelemetryProperties } from 'logger';
+import { ProcessEntryPointBase } from 'service-library';
+import { ApiController } from './controllers/api-controller';
+import { Newable, webApiIocTypes } from './setupIoContainer';
+
+// tslint:disable: no-any no-unsafe-any
+
+export class ControllerDispatcher extends ProcessEntryPointBase {
```

Comment:
> ControllerDispatcher extends ProcessEntryPoi [](start = 12, length = 45)

the way this is implemented, looks like you cannot have more than 1 public controller method

Reply:
yes this is intentionally